gcp-csm-observability/src/main/java/io/grpc/gcp/csm/observability/MetadataExchanger.java

```diff
@@ -280,8 +280,8 @@ private void setExchange(Metadata metadata) {
 
       @Override
       public void addLabels(AttributesBuilder to) {
-        put(to, "csm.service_name",      serviceName);
-        put(to, "csm.service_namespace", serviceNamespace);
+        put(to, "csm.service_name",           serviceName);
+        put(to, "csm.service_namespace_name", serviceNamespace);
```

Comment:
For consistency, can we rename the variable i.e s/serviceNamespace/serviceNamespaceName ?

Reply:
Well, it is still matching the filter metadata name. They can't both match. Seems it is no better one way vs the other.